README.md

```diff
@@ -97,11 +97,10 @@ header('Content-Type: application/json');
  
 // Try to do the POST with orderRequest Model.
 try {
-    $postResponse = $divideIq->request('stockbase_orderrequest', $orderRequest->toArray(), 'POST');
+    $postResponse = $divideIq->request('stockbase_orderrequest', $orderRequest->jsonSerialize(), 'POST');
```

Comment:
The `jsonSerialize()` method is called when you call `json_encode()` on an object. So the following are equivalent:

```php
$orderRequest->jsonSerializable();
json_encode($orderRequest);
```

So since you already call json_encode [here](https://github.com/Nav-Appaiya/PHPDivideIQ/blob/e684ca9f1e8a3a39f67f7236e78b1174ac5dfbc8/src/DivideIQ.php#L178), you can just pass the object as is.